# Material and Methods

**This is an example of a possible Material & Methods section**

Protein arrays were performed on 294 plasma and 92 serum samples from active vasculitis patients, healthy controls and different disease controls. Concentrations of 181 proteins were assessed by proximity extension assay (Olink Bioscience, Sweden) using the Inflammation and Cardiovascular III panel. Data are presented as normalized protein expression values called NPX, Olink Proteomicsâ€™ arbitrary unit on a log2 scale. In addition, 4 proteins were measured in plasma samples using the Luminex technique. analyses were performed for plasma and serum samples separately.

Univariate differential expression was performed on the NPX values using Olink's OlinkAnalyze R package (v1.2.4) using the olink_anova_posthoc function which performs an ANOVA for each protein followed by an emmeans post hoc analysis. This model was adjusted using age of patient and ckd_epi as covariates. P-values are adjusted using Tukey's procedure.
Multivariate analysis was performed using the PLS-DA approach as implemented in the R package mixOmics (v6.14.1). Internal validation was performed using 10 repeated 5 fold cross validations.

Gene set enrichment analysis was performed using the R package clusterProfiler (v3.18.1) using log2(fold change) as metric and a collection of gene sets including GO, KEGG and MsigDB (full collection can be found at http://download.baderlab.org/EM_Genesets/August_01_2020/Human/symbol/Human_GOBP_AllPathways_no_GO_iea_March_01_2020_symbol.gmt). Network enrichment analysis was performed using the R package neat (v1.2.3). All significantly upregulated and/or downregulated proteins (adjusted p-value < 0.05) were used as regulated gene sets. As gene set collection, the MSigDB signatures were extracted from the same gene set collection used for the GSEA. A collection of protein interactions were downloaded from the STRING DB (https://stringdb-static.org/download/protein.links.v11.0/9606.protein.links.v11.0.txt.gz). 

Correlations between NPX values and crp, sr and bvas scores were calculated using the Pearson correlation. Benjamini-Hochberg adjusted p-values were calculated to account for multiple testing.

Heatmaps were produced using the R package ComplexHeatmap (v3.18.1) and when appropriate complete clustering was performed on the euclidean distance.

The code used to produce all results can be found [here](https://raw.githubusercontent.com/vincent-van-hoef/Analysis5204/master/analysis/analysis.R).