---
title: Identification of clinical biomarkers for ANCA-associated vasculitis (#5204)
documentclass: book
link-citations: yes
favicon: image/favicon.ico
description: Report summarizing the results for project #5204
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  echo=FALSE,
  comment = "#>"
)
library("knitr")
```

# Introduction

The ANCA-associated vasculitides are rare but severe autoimmune disorders.
Tools for improved diagnostics and prognostics are needed. In this project, 184
proteins (Olink; CARDIOVASCULAR III and INFLAMMATION panels) and four
additional proteins (Luminex data) were measured in serum or plasma samples
from patients with ANCA-associated vasculitis and several controls (healthy,
remission, disease control), in order to identify biomarkers for active
disease, organ involvement and accurate diagnosis.

To accomplish this, following analyses have been agreed upon:

* Quality Control of PEA data
* Unsupervised analysis of data structure using PCA
* Differential expression of first set of samples comparing selected subgroups:
  - Active AAV vs Healthy
  - Active AAV vs Remission AAV
  - Active AAV vs Disease Control
  - GPA vs MPA
  - MPO-ANCA+ vs PR3-ANCA+
  - (Comparison between affected organs)
* Serum and plasma Samples will be analyzed separately
* Pathway analysis of differential expressed proteins using KEGG, GO, MsigDB, ... gene set collections
* Correlation of proteins with the inflammation markers and BVAS scores
* Effect of cortisone treatment on protein expression
* Biomarkers/Predictors of GPA vs MPA and MPO-ANCA+ vs PR3-ANCA+ (using appropriate ML techniques)

:::note
This report describes the performed analyses and a selection of the resulting
output of the plasma results only. The full results and all plots are found in
the accompanying Results folder.
:::

<!--chapter:end:index.Rmd-->

# Data Cleaning

Prior to analysis, the raw clinical data and expression data has to be cleaned; i.e sample names have to be consistent, spaces removed, all samples need metadata, ...

## Clinical Data

The metadata for the separate subgroups/diseases was combined into a single metadata dataframe per tissue. Column names were cleaned and made consistent between the different subgroups to allow for easier handling in R. Further cleaning of the dates was performed in R using the *lubridate* package and annotations such as age, time in freezer, BVAS score and site of sample origin were added as well.

Samples without Olink expression data were removed from the metadata of plasma ("lun1065") and serum (("lin113", "sle_121/a" and "sle_156/a").

The resulting metadata for serum and plasma separately is saved in the folder
**Results/Plasma/plasma_metadata.csv** and **Results/Serum/serum_metadata.csv**
respectively.

View the code for the preprocessing of the metadata of the
[plasma](https://raw.githubusercontent.com/vincent-van-hoef/Analysis5204/master/data-raw/plasma_metadata.R)
and the
[serum](https://raw.githubusercontent.com/vincent-van-hoef/Analysis5204/master/data-raw/serum_metadata.R)
by clicking on the links.

## Expression Data

The names of the samples were cleaned and modified to correspond to the metadata.
Three samples that were present in the expression data but had no metadata were
removed ("lun1059", "iga2156" and "umu78").

Three proteins (MCP-1, OPG and uPA) are measured in both the Cardiovascular and
the Inflammation panel. The expression values correlate well between the panels
so only the Inflammation data is kept for these proteins.

View the code for the preprocessing of the experimental data of the
[plasma](https://raw.githubusercontent.com/vincent-van-hoef/Analysis5204/master/data-raw/plasma_npx.R)
and the
[serum](https://raw.githubusercontent.com/vincent-van-hoef/Analysis5204/master/data-raw/serum_npx.R)
by clicking on the links.

In addition, all data and metadata can also be accessed via R:

```{r, eval = FALSE, echo=TRUE}
# install.packages("devtools")
devtools::install_github("vincent-van-hoef/Analysis5204")
library("Analysis5204")
data(list=c("plasma_metadata", "plasma_npx", "serum_metadata", "serum_npx", "gsea_sets", "lum_dat"),
      package = "Analysis5204")
```

# Quality Control

One additional sample (vaska637) is filtered out beforehand due to technical concerns (failed sample according to Olink QC rapport).

:::fyi
Shown in this section are the results for the plasma samples. The QC of the serum samples is available in the folder /Serum/QC/
:::

## Data Distribution

A general overview of the data distribution per plasma sample per panel, and
ordered by the median value on that panel is shown in Figure
\@ref(fig:plDistrBox). They show one boxplot per sample and can be colored by
any feature, but by default are colored by whether the sample passed technical
QC (“QC_Warning” status as defined by Olink). The things to look for in this
plot are odd distributions, deviant medians and patterns among the outlier
assays. Overall, distributions are fairly similar and the medians are
comparable. Two samples in the INFLAMMATION panel are deemed to have failed the
technical QC, but a further investigation has to show whether to remove them
from the dataset.

```{r plDistrBox, out.width="49%",out.height="49%",fig.show='hold', fig.align='center', fig.cap="Boxplots of NPX distribution per sample by panel ordered by median NPX value", fig.subcap=c("Cardiovascular III", "Inflammation")}
knitr::include_graphics(c("../Results/Plasma/QC/plasma_CVIII_dist.png",
"../Results/Plasma/QC/plasma_INF_dist.png"))
```

The distribution plot can get difficult to read when there are a large number
of samples or many panels. Plotting the interquartile range (IQR) vs the median
shows an alternative view on the data distributions (Figure \@ref(fig:plIQR)).
Most samples cluster together, some samples have a slightly more extreme IQR.
Even though two samples are marked with QC warnings, their data distributions
(and PCA plot - data not shown) look ok. According to the Olink data analysis
tutorial, this means the samples can be kept for downstream analysis so they
were retained for further analysis.

```{r plIQR, fig.show='hold', out.width="100%", fig.align='center', fig.cap="Interquartile range versus median per sample by panel ordered by median NPX value. Dashed lines indicate 3 standard deviations from the median or IQR mean."}
knitr::include_graphics(c("../Results/Plasma/QC/plasma_qc.png"))
```

## PCA

Using [Principal Component
Analysis](http://setosa.io/ev/principal-component-analysis) (PCA), we examine
the structure of the data by projecting the samples on a two-dimensional graph
using the two first principal components (PC). These components are linear
combinations of the proteins that explain the most variation between those
samples. These plots can be used to examine the samples for outliers, sample
swaps, batch effects and other relationships. When normalization successfully
removed technical or batch artefacts, the relative distances should be
biologically interpretable. A plot of PC1 vs PC2 using the 100 most variable
proteins and colored by their biological/experimental group is shown in Figure
\@ref(fig:plPCA). Each point corresponds to a sample plotted by PC1 and PC2.
The different experimental groups do seem to differ, especially along PC1.
Also, the healthy control samples seem to cluster more together than the active
or other control samples.

```{r plPCA, fig.show='hold', out.width="100%", fig.align='center', fig.cap="PCA plot of all plasma samples coloured by biological group."}
knitr::include_graphics(c("../Results/Plasma/QC/plasma_pca_100.png"))
```

In the QC folder you will find additionally a PCA plot of PC1 vs PC3, a
screeplot showing the explained variation per PC and a barplot showing the top
and bottom genes contributing to the variation on PC1.

<!--chapter:end:quality.Rmd-->

# Differential Expression

To compare the different sample groups against each other we applied two
complementary statistical techniques: univariate ANOVA and multivariate pls-da.
ANOVA (followed by a post-hoc analysis) has as a benefit that it is fairly
straightforward to interpret the results and will return the most differential expressed
proteins and how much they differ between the groups. The multivariate pls-da
technique is harder to interpret but can take correlations among the proteins
into account and might therefore result in a more robust set of proteins to
serve as markers distinguishing the groups. It will however not return an
estimate of the different expression among the groups, but rather a measure of
how important a certain protein is in separating the subgroups.

Following comparisons were looked at:

* aav_remission vs active_disease
* aav_remission vs healthy_controls
* aav_remission vs RA
* aav_remission vs SLE_nephritis
* active_disease vs healthy_controls
* active_disease vs RA
* active_disease vs SLE_nephritis
* healthy_controls vs RA
* healthy_controls vs SLE_nephritis
* MPA vs GPA
* MPO_pos vs PR3_pos
* RA vs SLE_nephritis

:::note
All results for each comparison are collected in either **/Plasma/Comparisons**
or **/Serum/Comparisons**.

To keep the size of this report manageable, the results shown here are limited to the
active_disease vs healthy_control comparison in Plasma. The results for all
other comparisons are found in the appropriate folder of the Results
directory.
:::

## Univariate Analysis

The ANOVA F-test will be used to assess whether the protein expression varies
significantly across the different sample groups, one protein at a time. If
this is the case, a post-hoc test will look at all pairwise group comparisons
to determine which comparisons are in fact resulting in the differential
expression. An adjusted p-value (Tukey) will be calculated per protein.

For the ANOVA analysis, a correction for the patient covariates *age* and
*ckd_epi* level was applied to the linear model.

Results for the per-protein analysis is collected in **Results/Plasma/Comparisons/{comparison}}/Univariate/** and **Results/Serum/Comparisons/{comparison}}/Univariate/**.

### Protein-level

A volcano plot of the post-hoc results is made for each comparison.  In these type of plots, proteins in the left or right upper quadrant are the most significantly differentially expressed. The volcano plot for active_disease vs healthy_control is shown in Figure \@ref(fig:glplots) (Left). In addition, a heatmap of proteins with a adjusted p-value < 0.05 is produced. In this visualization, samples are clustered according to the expression of these significant proteins (see Figure \@ref(fig:glplots) (Right)).

```{r glplots, fig.show='hold', out.width="49%", fig.align='center', fig.cap="Two visualizations of the gene-level differential expression. Volcano plot (Left) Heatmap of the significant proteins (Right)."}
knitr::include_graphics(c("../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/active_disease_vs_healthy_controls_volcano_anova_posthoc.png", "../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/Heatmap_Sig_Prots_Prots.png"))
```

The top 15 of most significantly regulated proteins (ranked by adjusted p-value) of active_disease versus healthy_controls is shown in \@ref(fig:top15). The "estimate" column can be considered as a log2 fold change, where a higher estimate means a higher expression in active_disease samples (i.e. a difference of 1 indicates a doubled expression).

```{r top15, fig.show='hold', out.width="75%", fig.align='center', fig.cap="Table of 15 most DE proteins"}
knitr::include_graphics(c("../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/active_disease_vs_healthy_controls_Top15_table.png"))
```

A full list of results for all proteins can be found in **/Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/active_disease_vs_healthy_controls_anova_posthoc_results.xlsx**.

### GSEA

Each univariate comparison folder (e.g. **Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/**) contains a **GSEA/** folder that contains the output of a gene set enrichment analysis. GSEA uses a  statistical metric comparing two conditions to rank all genes (for example according to Log2FoldChange) and applies a weighted Kolmogorov–Smirnov (KS) statistic to calculate an Enrichment Score (ES).

:::fyi
Calculation of the Enrichment Score:

* Rank genes by a statistic (e.g. log2FoldChange)
* Compute cumulative sum over ranked genes:
  - Increase sum when gene in set, decrease it otherwise. 
  - Magnitude of increment depends on correlation of gene
with phenotype.
* Record the maximum deviation from zero as the enrichment score (ES)
:::

The ES and NES (ES normalized for set size) of a set are calculated by looking at where the statistics of genes belonging to a certain set can be found in the ranked gene list. A high (N)ES indicates these genes are found high up in the list. In other words, a high (N)ES value means that for the genes in this set there is - on average - a shift towards a higher expression. The significance of each (N)ES is calculated permuting the sets and recomputing ES, getting a null distribution for the ES. A multiple comparison correction is also performed on the p values. More info on the GSEA algorithm can be found [here](https://bioinformatics.mdanderson.org/MicroarrayCourse/Lectures09/gsea1_bw.png).

In this project we used the "estimate" as the statistic to rank the genes (equivalent to log2FoldChange). When it comes to the gene sets to be tested, a compendium of gene sets developed and maintained by the Bader Lab was used (called Human_GOBP_AllPathways_no_GO_iea_March_01_2021_symbol.gmt; download [here](http://download.baderlab.org/EM_Genesets/March_01_2021/Human/symbol/Human_GOBP_AllPathways_no_GO_iea_March_01_2021_symbol.gmt)). This gene set collection combines data from Gene Ontology (GO) with pathway data from MsigdB, Reactome, WikiPathways, ... for a total of around 18500 gene sets.

Each GSEA results folder contains a .xlsx file containing the full results and two different visualizations of the top terms (see Figure \@ref(fig:gseaPlots)). These figures include a barplot of the gene sets with the 6 highest and 6 lowest NES values and an Enrichment map of the 20 gene sets with the highest absolute NES values. Enrichment maps organize gene sets into a network with edges connecting overlapping gene sets. In this way, mutually overlapping gene sets tend to cluster together, making it easier to identify functional modules. 

```{r gseaPlots, out.width="49%", fig.cap="Example of the different GSEA visualizations. (Left)  Barplot of the gene sets with the 10 highest and 10 lowest NES values. (Right) Enrichment map of the same gene sets.",fig.show='hold',fig.align='center'}
knitr::include_graphics(c("../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/GSEA/active_disease_vs_healthy_controls_barplot.png", "../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/GSEA/active_disease_vs_healthy_controls_emap.png"))
```

In addition, in the **GSEA_PLOTS/** folder you can find a visualization of the location of the gene set genes in the ranked gene list and an ES calculation visualization (see Figure \@ref(fig:gseaNes)).

```{r gseaNes, out.width="49%", fig.cap="Example of two ES visualizations and calculation. (Left) Downregulation of Notch signaling. (Right) Upregulation of protein modification.",fig.show='hold',fig.align='center'}
knitr::include_graphics(c("../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/GSEA/GSEA_PLOTS/active_disease_vs_healthy_controls_gseaplot_SIGNALING_BY_NO.png", "../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/GSEA/GSEA_PLOTS/active_disease_vs_healthy_controls_gseaplot_POST-TRANSLATIO.png"))
```

### NEA

NEA or Network Enrichment Analysis is an alternative method to asses which biological functions are regulated following an experiment or when comparing different conditions. As opposed to GSEA, where you look at the overlap of genes in a ranked list and in a gene set directly, in NEA you take the **number of known associations** between genes found to be significant and a gene set into account. To do this we use the NEA algorithm as implemented in the NEAT R package (see [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1203-6) for a relatively easy explanation).

:::fyi
Methods paper for NEAT (NEA Test):

The starting point of enrichment analyses is the identification of one or more gene sets of interest. These target gene sets are typically groups of genes that are differentially expressed between experimental conditions, but they can also be different types of gene sets: e.g., clusters of genes that are functionally similar in a given time course, or genes that are bound by a particular protein in a ChIP-chip or ChIP-seq experiment. Enrichment analysis provides a characterization of each target gene set by testing whether some known functional gene sets can be related to it. Methods for gene enrichment analysis assess the relationship between a target gene set and each functional gene set simply by considering the overlap of these two groups. In contrast to this, network enrichment analysis incorporates an evaluation of the level of association between genes in the target set and genes in the functional gene set into the test.
:::

The significant genes were defined here as to have an adjusted p-value smaller than 0.05 and an estimate of either larger than 0 ("upregulated genes") or smaller thn 0 ("downregulated genes"). The genes set collection consisted of a slimmed down version of the gene set collection used for GSEA (only MSigDB terms for computational resons). The known associations were collected from [stringDB](https://string-db.org).

Each univariate comparison folder (e.g. **Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Univariate/**) contains a **NEA/** folder that contains the output of the network enrichment analysis. This output consists of 2 excel files: one containing the results for the upregulated genes (e.g. active_disease_vs_healthy_controls_neat_up_results.xlsx) and one containing the results for the downregulated genes (e.g. active_disease_vs_healthy_controls_neat_down_results.xlsx).

## Multivariate analysis

In addition, a multivariate technique called PLS-DA was used. Information about this method can be found [here](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-253) and [here](http://mixomics.org/case-studies/splsda-srbct/). This method attempts to represent the data in a lower dimensional set of "components", similarly to PCA. Unlike PCA, which attempts to retain as much variation of one dataset in the components, PLS methods try to maximize the covariance between two datasets (here the expression data and the phenotype data). In this way, the phenotype separation will be maximized. PLS-DA attempts to find the most optimal way to represent multidimensional data taking sample classes into account using as little components as possible. 

To better understand how this "black box" model works, the samples are visualized in the reduced PLS space (see Figure \@ref(fig:plsPlot) (Left)). In this plot healthy controls separate quite well from the active disease samples on - mainly - Component 1. By looking at which proteins contribute the strongest to Component 1 (see Figure \@ref(fig:plsPlot) (Right)), one can get an idea of which proteins are most important in separating these two phenotypes. The proteins with the highest absolute loadings will drive the separation the most. For example, a protein with a high positive loading on component 1 will contribute significantly to the separation on component 1. Moreover, a **positive** loading value on component 1 indicates that the expression of that protein is also higher in samples that have a higher component 1 value. a **negative** loading value on component 1 indicates that the expression of that protein is lower in samples that have a higher component 1 value. Looking at and combining these plots, one can get an idea on how the samples are separated and which proteins are important to drive this separation.

```{r plsPlot, out.width="49%", fig.cap="PLS-DA visualization. (Left) Samples projected into the subspace spanned by the first two PLS components. (Right) top 15 of loading weights on Component 1, color indicates the class for which the selected variable has a maximal mean value.",fig.show='hold',fig.align='center'}
knitr::include_graphics(c("../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Multivariate/active_disease_vs_healthy_controls_SamplePlot_2comp.png", "../Results/Plasma/Comparisons/active_disease_vs_healthy_controls/Multivariate/active_disease_vs_healthy_controls_Loadings_Comp1_Top15.png"))
```

Results for the PLS-DA analysis are collected in **Results/Plasma/Comparisons/{comparison}}/Multivariate/** and **Results/Serum/Comparisons/{comparison}}/Multivariate/**.

The loading weights for all proteins on the first 5 components can be found in in an excel file (e.g. active_disease_vs_healthy_controls_Loadings_Scores.xlsx). In the same folder, visualization of the samples in the PLS subspace spanned by the first 2 components, the top 15 loading weights on Comp 1 and 2 and a biplot showing the samples nd most significant proteins on the same plot are collected.

<!--chapter:end:de.Rmd-->

# Correlation

Pearson correlations of each protein versus crp, sr and bvas values were calculated.

## crp

The full list of protein-wise pearson correlation results with crp is collected in excel files in **Results/Plasma/Correlation/crp/crp_pearson_correlation.xlsx** and **Results/Serum/Correlation/crp/crp_pearson_correlation.xlsx**

All individuals regressions are plotted and collected in the subfolder **regression_plots/** (see Figure \@ref(fig:crpCorr))

```{r crpCorr, fig.show='hold', out.width="49%", fig.align='center', fig.cap="Most significant positively crp correlated protein (Left) Most significant negatively crp correlated protein (Right). Indicated in red is the pearson correlation coefficient and 95% confidence interval in grey"}
knitr::include_graphics(c("../Results/Plasma/Correlation/crp/regression_plots/IL6_regression.png", "../Results/Plasma/Correlation/crp/regression_plots/SCF_regression.png"))
```

## sr

The full list of protein-wise pearson correlation results with sr is collected in excel files in **Results/Plasma/Correlation/sr/sr_pearson_correlation.xlsx** and **Results/Serum/Correlation/sr/sr_pearson_correlation.xlsx**

All individuals regressions are plotted and collected in the subfolder **regression_plots/** (see Figure \@ref(fig:srCorr))

```{r srCorr, fig.show='hold', out.width="49%", fig.align='center', fig.cap="Most significant positively sr correlated protein (Left) Most significant negatively sr correlated protein (Right). Indicated in red is the pearson correlation coefficient and 95% confidence interval in grey"}
knitr::include_graphics(c("../Results/Plasma/Correlation/sr/regression_plots/CCL23_regression.png", "../Results/Plasma/Correlation/sr/regression_plots/SCF_regression.png"))
```

## bvas

The full list of protein-wise pearson correlation results with bvas is collected in excel files in **Results/Plasma/Correlation/bvas/bvas_pearson_correlation.xlsx** and **Results/Serum/Correlation/bvas/bvas_pearson_correlation.xlsx**

All individuals regressions are plotted and collected in the subfolder **regression_plots/** (see Figure \@ref(fig:bvasCorr))

```{r bvasCorr, fig.show='hold', out.width="49%", fig.align='center', fig.cap="Most significant positively bvas correlated protein (Left) Most significant negatively bvas correlated protein (Right). Indicated in red is the pearson correlation coefficient and 95% confidence interval in grey"}
knitr::include_graphics(c("../Results/Plasma/Correlation/bvas/regression_plots/GDF-15_bvas_correlation.png", "../Results/Plasma/Correlation/bvas/regression_plots/IL-1 alpha_bvas_correlation.png"))
```

<!--chapter:end:correlation.Rmd-->

# Cortisone

## Protein-level

Active disease samples were split in an experimental group that was treated with cortisone and a control group that was not treated with cortisone. Differential analysis between both groups was calculated as in Section 4.1; i.e. per-protein anova on *age* and *ckd_epi* corrected data followed by a post-hoc analysis.

Results for the per-protein analysis is collected in **Results/Plasma/Cortisone** and **Results/Serum/Cortisone**.

The volcano plot for cortisone vs no_cortisone is shown in Figure \@ref(fig:cortplots) (Left).

```{r cortplots, fig.show='hold', out.width="100%", fig.align='center', fig.cap="Volcano plot of cortisone versus no cortisone. Horizontal line indicates an adjusted p-value of 0.05"}
knitr::include_graphics(c("../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/Cortisone_vs_No_Cortisone_volcano_anova_posthoc.png"))
```

The top 15 of most significantly regulated proteins (ranked by adjusted p-value) of cortisone versus no_cortisone is shown in \@ref(fig:top15Cort). The "estimate" column can be considered as a log2 fold change (i.e. a difference of 1 indicates a doubled expression).

```{r top15Cort, fig.show='hold', out.width="75%", fig.align='center', fig.cap="Table of 15 most DE proteins"}
knitr::include_graphics(c("../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/Cortisone_vs_No_Cortisone_Top15_table.png"))
```

## GSEA

GSEA was performed for cortisone versus no_cortisone as in Section 4.1.2.

As before, the GSEA results folder in the Cortisone folder contains a .xlsx file containing the full GSEA results and two different visualizations of the top terms (see Figure \@ref(fig:gseaPlotsCort)). These figures include a barplot of the gene sets with the 6 highest and 6 lowest NES values and an Enrichment map of the 20 gene sets with the highest absolute NES values. Enrichment maps organize gene sets into a network with edges connecting overlapping gene sets. In this way, mutually overlapping gene sets tend to cluster together, making it easier to identify functional modules. 

```{r gseaPlotsCort, out.width="49%", fig.cap="Example of the different GSEA visualizations for cortisone versus no cortisone in Plasma. (Left)  Barplot of the gene sets with the 6 highest and 6 lowest NES values. (Right) Enrichment map of the 20 gene sets with the highest absolute NES values.",fig.show='hold',fig.align='center'}
knitr::include_graphics(c("../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/Cortisone_vs_No_Cortisone_barplot.png", "../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/Cortisone_vs_No_Cortisone_emap.png"))
```

In addition, in the **Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/GSEA_PLOTS** and **Results/Serum/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/GSEA_PLOTS**folder you can find a visualization of the location of the gene set genes in the ranked gene list and an ES calculation visualization (see Figure \@ref(fig:gseaNesCort)).

```{r gseaNesCort, out.width="49%", fig.cap="Example of two ES visualizations and calculation. (Left) Downregulation of cell proliferation. (Right) Upregulation of proteolysis.",fig.show='hold',fig.align='center'}
knitr::include_graphics(c("../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/GSEA_PLOTS/Cortisone_vs_No_Cortisone_gseaplot_POSITIVE_REGULA.png", "../Results/Plasma/Cortisone/Cortisone_vs_No_Cortisone/Univariate/GSEA/GSEA_PLOTS/Cortisone_vs_No_Cortisone_gseaplot_PROTEOLYSIS.png"))
```

<!--chapter:end:cortisone.Rmd-->

# Data Responsibility

- **NBIS & Uppnex** Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.
- **Sensitive data** Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.
- **Long-term backup** The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

<!--chapter:end:data_responsibility.Rmd-->

# Acknowledgements

If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.

- **NBIS Staff** are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. `ICMJE recommendations <http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html>`_). If applicable, please include **Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University**, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: `"Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged"	<https://www.nbis.se/resources/support.html>`_.

- **UPPMAX** If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: `"The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ"	<https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/>`_.

- **NGI** In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: `"The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure"	<https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication>`_.

<!--chapter:end:acknowledgements.Rmd-->

# Closing Procedures

You should soon be contacted by one of our managers, [Jessica Lindvall](jessica.lindvall@nbis.se) or [Henrik Lantz](henrik.lantz@nbis.se), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within **30 days** the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via [NBIS support](http://nbis.se/support/support.html).

Thank you for using NBIS and all the best for future research.

<!--chapter:end:closing.Rmd-->

